docker/node-wasm

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+path="$(dirname $1)"
+file="$(basename $1)"
+
+# Workaround for
+# https://github.com/kripken/emscripten/issues/4542
+
+if [ "$(basename $path)" != "deps" ]; then
```

Comment:
Could you add a comment here indicating what kind of paths we expect to see here and why we have to branch here and in the if below?